GETTING_STARTED.md

```diff
@@ -46,16 +52,19 @@ __Florence steps:__
 * [Maven](https://maven.apache.org/)
 * [Docker](https://www.docker.com/get-started)
 * [Node.js and npm](https://nodejs.org/en/), known working versions:
-  - Node.js version 8.9.4
-  - npm version 5.6.0
+  - Node.js version 10.15.3
+  - npm version 6.9.0
 * [dp-compose](https://github.com/ONSdigital/dp-compose)
   - Elasticsearch 2.4.2
   - Highcharts
   - Postgres
 * [go](https://golang.org/doc/install)
+* [GoConvey](https://github.com/smartystreets/goconvey#installation)
+* [Govendor](https://github.com/kardianos/govendor)
 
 For CMD additionally install:
 
+* [Zookeeper](https://formulae.brew.sh/formula/zookeeper)
```

Comment:
Brew installing zookeeper and then following the Kafka quickstart guide will result in 2 instances of zookeeper running. Perhaps we should update the Kafka link to also the brew installed Kafka? or we dont need to add a zookeeper link

Reply:
Removing zookeeper link